src/MySQLReplication/Event/DTO/RowsQueryDTO.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MySQLReplication\Event\DTO;
+
+use MySQLReplication\Definitions\ConstEventsNames;
+use MySQLReplication\Event\EventInfo;
+use function get_object_vars;
+use const PHP_EOL;
```

Comment:
```suggestion
```

Reply:
Right now you can run `composer sa` its will check phpstan and ecs but yes its not added to gitci and also I cant find rule to check that.